airflow/contrib/hooks/spark_submit_hook.py

```diff
@@ -255,12 +254,11 @@ def _process_log(self, itr):
 
     def on_kill(self):
         if self._sp and self._sp.poll() is None:
-            logging.info('Sending kill signal to spark-submit')
-            self.sp.kill()
+            logging.info('Sending kill signal to %s' % self._connection['spark_binary'])
+            self._sp.kill()
 
             if self._yarn_application_id:
                 logging.info('Killing application on YARN')
```

Comment:
I believe it's kinda similar, only adding extra code.
Do you really want me to update with the code above, and to add some tests or it's fine this way ?

thx